packages/shared/components/popups/BalanceFinder.svelte

```diff
@@ -33,9 +33,9 @@
                 return
             }
 
-            await asyncSyncAccounts(addressIndex, gapIndex, accountDiscoveryThreshold, false)
+            await asyncSyncAccounts(addressIndex, gapLimit, accountDiscoveryThreshold, false)
 
-            addressIndex += gapIndex
+            gapLimit+=gapLimit
```

Comment:
We should make this increment by a fixed constant, so that the gap limit increments linearly and not exponentially.  Currently we will end up with a huge gapLimit just to search for more accounts.